packages/preview-button/admin/src/utils/index.js

```diff
@@ -1 +1,4 @@
+export { default as getTrad } from './getTrad';
```

Comment:
Zou het niet makkelijker zijn om minder default exports te gebruiken, zodat je gewoon dit kan doen? Default exports voelen voor mij ook niet heel schaalbaar; wat als je een tweede functie wil toevoegen aan het bestand, wordt het dan niet onhandig inconsistent?

```suggestion
export { getTrad } from './getTrad';
```